src/transaction/log_replication.cpp

```diff
@@ -518,7 +518,12 @@ namespace cublog
       }
 
     btree_root_update_stats (&thread_entry, root_page, stats);
-    pgbuf_set_lsa (&thread_entry, root_page, &record_lsa);
+
+    // it is not needed to update page LSA for btree root statistics
+    // if this is done, in scalability scenarios with local storage, the LSA's of active transaction server's
+    // and page server's pages will differ because of this update
```

Comment:
I don't think is correct. If the system crashes and restarts, I guess that unique stats are wrong. You may check that the difference is unique stats and ignore it.

Reply:
Refactored to not compare btree root pages' LSAs